.github/workflows/test.yml

```
@@ -25,7 +25,7 @@ jobs:
           path: tmp
 
       - name: Set up Node.js
-        uses: actions/setup-node@v2.5.0
+        uses: actions/setup-node@v2.5.1
```

Unpin to reduce noise
```suggestion
        uses: actions/setup-node@v2
```

---

If you are interested https://github.com/nodejs/remark-preset-lint-node/compare/main...XhmikosR:patch-1